build/post-app.js

```diff
@@ -14,6 +14,11 @@ exports.execute = function(options, webapp) {
 
   nodeHelper.require('copy-build-stage-data', options);
 
+  // If we need to transform code for Raptor tests
+  if ('1' === options.RAPTOR_TRANSFORM) {
+    require('./raptor-test-transformer').execute(options, webapp);
```

Comment:
We've already assign webapp to options at https://github.com/snowmantw/gaia/blob/raptoraop/build/post-app.js#L7
so you can access it within the options object